dev/content/Forms.vue

```diff
@@ -190,6 +199,7 @@ const toggleProps = [{ name: "modelValue", required: true, type: "string" }]
           <div class="mt-1">
             <form @submit.prevent>
               <BaseInput
+                v-model="inputVals['baseInput']"
```

Comment:
This highlights the primary value of allowing undefined v-model values.  Rather than needing to explicitly type your payload model, you should be able to do this instead:

```vue
<script setup lang="ts">
// won't require you to set every key, but will only allow keys currently on User 
const formVals = ref<Partial<User>>({})
</script>

<template>
    <BaseInput v-model="formVals['email']" type="email" label="Email" required />
</template>
```

It will be a bit experimental to see whether this is a good thing, but it's ultimately opt in.